projects/ngrx.io/content/guide/store/action-groups.md

```diff
@@ -0,0 +1,67 @@
+# Action Groups
+
+Action groups is a feature that allows you to group actions together. While writing actions, the actions in most of the cases looks like below.
+
+<code-example header="products-page.actions.ts">
+import { createAction, props } from '@ngrx/store';
+// defining an action without payload
+export const opened = createAction('[Products Page] Opened');
+// defining an action with payload using the `props` function
+export const paginationChanged = createAction(
+  '[Products Page] Pagination Changed',
+  props&lt;{ page: number; offset: number }&gt;()
+);
+// defining an action with payload using the props factory
+export const queryChanged = createAction(
+  '[Products Page] Query Changed',
+  (query: string) => ({ query })
+);
+</code-example>
+
+
+With the help of this new `createActionGroup` api, we can avoid the above code and write the following code.
+
+<code-example header="products-page.actions.ts">
+
+import { createActionGroup, emptyProps, props } from '@ngrx/store';
+export const ProductsPageActions = createActionGroup({
+  source: 'Products Page',
+  events: {
+    // defining an event without payload using the `emptyProps` function
+    'Opened': emptyProps(),
+    
+    // defining an event with payload using the `props` function
+    'Pagination Changed': props&lt;{ page: number; offset: number }&gt;(),
+    
+    // defining an event with payload using the props factory
+    'Query Changed': (query: string) => ({ query }),
+  }
+});
+</code-example>
+
+After moving to the new `createActionGroup` api, we can use the following code to import and dispatch the action.
+
+```typescript
+
+import { ProductsPageActions } from './products-page.actions';
+// action type: [Products Page] Opened
+ProductsPageActions.opened();
+// action type: [Products Page] Pagination Changed
+ProductsPageActions.paginationChanged({ page: 10, offset: 100 });
+// action type: [Products Page] Query Changed
+ProductsPageActions.queryChanged('ngrx');
+
+```
+
+You dont need to use below syntax to import actions.
+
+```typescript 
+
+import * as productActions from './product.actions';
+
+```
```

Comment:
What do you mean here?
I think it can be removed.

```suggestion
You dont need to use below syntax to import actions.

```typescript 

import * as productActions from './product.actions';

```
```